l5kit/l5kit/data/zarr_utils.py

```diff
@@ -6,6 +6,7 @@
 from . import FRAME_DTYPE, SCENE_DTYPE, ChunkedDataset
 
 
+# TODO add testing
```

Comment:
Same here. Although here it seems like that will be more involved and need its own PR.

Reply:
yes this one is not trivial